### PR TITLE
Fix error caused by go-gogs-client, And inital Godeps commit

### PR DESCRIPTION
**Pull request by techzilla**
https://github.com/gogits/go-gogs-client/commit/9ea35aaa6631ad4f5ad37b6bf4f82de66023fed6

At least I think this identified change was the cause, and not just a symptom.

### CODE REVIEW

**Comment by techzilla** [comment]
OK, Finally.

This is about as good as I could get the build process, without using the rewrite imports flag.  The symlink fixed everything locally, but TravisCI choked on it...  

So, that means to use the godeps locally, right from repo, 

*First we clone*
    git clone ..../gitea
    cd gitea

*Then recreate that symlink, otherwise cmd will choke on import*
    mkdir Godeps/_workspace/src/github.com/go-gitea
    ln -s '../../../../..' Godeps/_workspace/src/github.com/go-gitea/gitea

*Finally Build*
    godep go build

Remember if you don't prefix your commands with godep, you won't use the checked in workspace.  
But that clone can now be placed anywhere on the filesystem, and is not chained to a perfect go path.

File: .travis.yml [hunk 1]
```diff
@@ -9,9 +9,9 @@ go:
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -y libpam-dev
-  - go get github.com/msteinert/pam
```
Comment (tboerger) [inline .travis.yml]: Does it work on non-linux machines without a problem to add the pam stuff to the deps?

**Comment by lunny** [comment]
So  many files changed.

**Comment by tboerger** [comment]
Do we really want to do that? Or do we want to switch to 1.5 builds only and use the new vendor functionality?

**Comment by lunny** [comment]
I think a Go 1.5 build only is interesting.

**Comment by tboerger** [comment]
Let's close that in favor of still using gopm until we can switch to go 1.5 vendoring.

**Pull request closed without merging**